sinch/core/enums.py

```diff
@@ -13,3 +13,4 @@ class HTTPAuthentication(Enum):
     BASIC = "BASIC"
     OAUTH = "OAUTH"
     SIGNED = "SIGNED"
+    SMS_TOKEN = "SMS_TOKEN"
```

Comment:
Maybe `BEARER_TOKEN` would be more generic as a name for a method that required a bearer token as a value for authentication

Reply:
IIRC both `OATUH` and `SMS_TOKEN` are `Bearer` tokens. That's why the naming.

Reply:
Well, from a code owner point of view, `BEARER_TOKEN` would make more sense as it says what it is. And if I understand correctly, behind the `OAUTH` keyword, there is not a direct `Bearer` with a static token but a logic that check for a JWT existence. So as this one is a standard `Bearer` token usage, it would make sense to refactor the HTTPTransport class to:
 - have a single implementation of `Authorization` header with a `Bearer` token formatting (as you are saying it's used in both OAUTH and SMS_TOKEN)
 - be able to support more APIs with static bearers instead of duplicating the code

Reply:
`Well, from a code owner point of view, BEARER_TOKEN would make more sense as it says what it is` - From my point of view my naming is much more clearer (subjective thing). Having `OATUH` and `SMS_TOKEN` as separate entities is much more descriptive and in reality `it says what it is`.
When it comes to the code duplication in HTTP transport:
```
        elif endpoint.HTTP_AUTHENTICATION == HTTPAuthentication.SMS_TOKEN.value:
            if not self.sinch.configuration.sms_api_token or not self.sinch.configuration.service_plan_id:
                raise ValidationException(
                    message=(
                        "sms_api_token and service_plan_id are required by this API. "
                        "Those credentials can be obtained from Sinch portal."
                    ),
                    is_from_server=False,
                    response=None
                )
            request_data.headers.update({
                "Authorization": f"Bearer {self.sinch.configuration.sms_api_token}",
                "Content-Type": "application/json"
            })
```
As you can see I use separate Enum values for two versions of Bearer authentication, simply to have code that is cleaner and more descriptive (specialised validation and error message).
When it comes to the code duplication:
```
            request_data.headers.update({
                "Authorization": f"Bearer {self.sinch.configuration.sms_api_token}",
                "Content-Type": "application/json"
            })
```
That's a simple dict `literal` (no logic here). I can move that out to a method or function but in reality the readability will be diminished (jumping across method/function calls). 3 lines of duplicated code is sometimes  a cleaner way.

btw. It looks like you're doing it similar thing, but on much bigger scale in your SDK:
Let's take example of [`packages/voice/src/rest/v1/conferences/conferences-api.ts](https://github.com/sinch/sinch-sdk-node/blob/main/packages/voice/src/rest/v1/conferences/conferences-api.ts#L48)

```
    const headers: { [key: string]: string | undefined } = {
      'Content-Type': 'application/json',
      'Accept': 'application/json',
    };
```
This dict/object literal is duplicated 3 times in one file.